### ISSUE TITLE
Titanic Classification

### ISSUE THREAD

**Issue by CHEVURIVENKATESH2709** [CHEVURIVENKATESH2709/CHEVURIVENKATESH#2]
from sklearn.linear_model import LogisticRegression
from sklearn.metrics import accuracy_score

# Build and train the model
model = LogisticRegression()
model.fit(X_train, y_train)

# Make predictions on the test set
y_pred = model.predict(X_test)

# Calculate accuracy
accuracy = accuracy_score(y_test, y_pred)
print(f"Accuracy: {accuracy:.2f}")

# Get feature importances (coefficients in the case of Logistic Regression)
feature_importances = model.coef_[0]
features = ['Pclass', 'Age', 'Sex']
importance_dict = dict(zip(features, feature_importances))

# Display feature importances
print("Feature Importances:")
for feature, importance in importance_dict.items():
    print(f"{feature}: {importance:.2f}")
